### PR TITLE
All contribot

### LINKED ISSUE #30: Add all-contributors bot
## Related to frontend or backend ? 
- [] frontend
- [] backend
- [x] devops 

## Summary

This is a good bot, could be helful

## Issue Type 

- [x] Repo enhancement

### PR DESCRIPTION
**Pull request by ajitesh13**
## Issue Fix
Fixes #30 

## Description
The all-contributor bot is helpful. 

## Reviewers
@agg-shambhavi  and @routrohan  please review this PR

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "code"
+      ]
+    },
+    {
+      "login": "agg-shambhavi",
+      "name": "Shambhavi Aggarwal",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/48705124?v=4",
+      "profile": "https://www.linkedin.com/in/shambhavi-aggarwal-437804179/",
+      "contributions": [
+        "code"
```
Comment (agg-shambhavi) [inline .all-contributorsrc:37]: Please add documentation and planning here.

File: .all-contributorsrc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "Ajitesh13",
+      "name": "Ajitesh Panda",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/42615579?v=4",
+      "profile": "https://bugzilla.mozilla.org/user_profile?user_id=650533",
+      "contributions": [
+        "code"
```
Comment (agg-shambhavi) [inline .all-contributorsrc:19]: Add Planning here.

File: .all-contributorsrc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      ]
+    },
+    {
+      "login": "routrohan",
+      "name": "Rohan Rout",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/32121879?v=4",
+      "profile": "https://github.com/routrohan",
+      "contributions": [
+        "code"
```
Comment (agg-shambhavi) [inline .all-contributorsrc:28]: Add planning here.

File: .all-contributorsrc (hunk 1 above)
Comment (ajitesh13) [reply .all-contributorsrc:37]: Oh, sorry I forgot! ok, Merge this one now, I am creating another issue to fix this

Comment (agg-shambhavi) [reply .all-contributorsrc:37]: Okay.

**Review by agg-shambhavi — APPROVED** [review]

**Pull request merged**